src/test/java/uk/gov/hmcts/reform/wacaseeventhandler/services/InitiationTaskHandlerTest.java

```diff
@@ -0,0 +1,69 @@
+package uk.gov.hmcts.reform.wacaseeventhandler.services;
+
+import lombok.Builder;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.hmcts.reform.wacaseeventhandler.clients.WaWorkflowApiClientToInitiateTask;
+import uk.gov.hmcts.reform.wacaseeventhandler.domain.EvaluateDmnResponse;
+import uk.gov.hmcts.reform.wacaseeventhandler.domain.initiatetask.InitiateTaskDmnResponse;
+import uk.gov.hmcts.reform.wacaseeventhandler.helpers.InitiateTaskHelper;
+import uk.gov.hmcts.reform.wacaseeventhandler.services.initiatetask.InitiationTaskHandler;
+
+import java.util.Collections;
+import java.util.stream.Stream;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@ExtendWith(MockitoExtension.class)
+class InitiationTaskHandlerTest {
+
+    @Mock
+    private WaWorkflowApiClientToInitiateTask apiClientToInitiateTask;
+
+    @InjectMocks
+    private InitiationTaskHandler handlerService;
+
+    @ParameterizedTest
+    @MethodSource(value = "scenarioProvider")
+    void can_handle(Scenario scenario) {
+
+        Mockito.when(apiClientToInitiateTask.evaluateDmn(
+            "getTask_IA_Asylum",
```

Comment:
Same as the earlier comment making this static and public will mean we can update the name in one place only :)

Reply:
Yes, I plan to do it as part of the last PR for this rwa-238 works.